kDrive/Utils/UploadCountManager.swift

```diff
@@ -41,11 +46,19 @@ final class UploadCountManager {
     init(driveFileManager: DriveFileManager, didUploadCountChange: @escaping () -> Void) {
         self.driveFileManager = driveFileManager
         self.didUploadCountChange = didUploadCountChange
+        uploadCountObserver = uploadCountSubject
+            .debounce(for: .seconds(1), scheduler: DispatchQueue.main)
```

Comment:
```suggestion
           .throttle(for: .seconds(1), scheduler: DispatchQueue.main, latest: true)
```
With debounce the value is never published

Reply:
Yep correct. nice catch 👍